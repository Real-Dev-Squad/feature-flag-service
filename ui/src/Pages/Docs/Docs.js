import Layout from "../../Components/Layout/Layout";

export default function Docs(){
    return(
        <Layout>
            <h1>Docs</h1>
        </Layout>
    )
}