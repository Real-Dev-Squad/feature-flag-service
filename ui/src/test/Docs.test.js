import { render } from '@testing-library/react';
import Docs from '../Pages/Docs/Docs';

describe('<Docs/> component', () => {
    it('component should render',() => {
        render(<Docs />);
    });

    it('snapshpt should match',() => {
        const { container } = render(<Docs />);
        expect(container).toMatchSnapshot();
    });
})