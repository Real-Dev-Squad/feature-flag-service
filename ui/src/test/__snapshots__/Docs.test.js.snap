// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Docs/> component snapshot should match 1`] = `
<div>
  <header
    class="py-2 px-8 h-20 flex items-center bg-rds-primary text-slate-100"
  >
    <img
      class="rounded-full h-12 block mx-2"
      src="https://res.cloudinary.com/realdevsquad/image/upload/v1649837643/profile/EYNHBSK7riBKY775OOMB/mfzbgspqgvj6unz34unm.jpg"
    />
    <h1
      class="text-lg"
    >
      Pallab Sonowal
    </h1>
    <button
      class="py-2 px-4 rounded-lg ml-auto bg-white text-rds-primary hover:bg-slate-400"
    >
      Logout
    </button>
  </header>
  <main
    class="flex flex-row my-4"
  >
    <nav
      class="px-3"
    >
      <ul
        role="list"
      >
        <li
          class="text-blue-600 hover:underline text-sm py-1"
          role="listitem"
        >
          <a
            aria-current="page"
            class="active"
            href="/"
          >
            Home
          </a>
           
        </li>
        <li
          class="text-blue-600 hover:underline text-sm py-1"
          role="listitem"
        >
          <a
            class=""
            href="/users"
          >
            Create Users
          </a>
           
        </li>
        <li
          class="text-blue-600 hover:underline text-sm py-1"
          role="listitem"
        >
          <a
            class=""
            href="/flags"
          >
            Flags
          </a>
           
        </li>
        <li
          class="text-blue-600 hover:underline text-sm py-1"
          role="listitem"
        >
          <a
            class=""
            href="/sdk"
          >
            SDK
          </a>
           
        </li>
        <li
          class="text-blue-600 hover:underline text-sm py-1"
          role="listitem"
        >
          <a
            class=""
            href="/docs"
          >
            Docs
          </a>
           
        </li>
      </ul>
    </nav>
    <section>
      <h1>
        Docs
      </h1>
    </section>
  </main>
  <footer
    class="my-8 w-screen flex justify-center text-xs"
  >
    <span
      class="ml-2"
    >
      Copyright 2022-2023
    </span>
    <a
      class="ml-2 hover:underline"
      href="http://www.google.com"
    >
      RDS Website
    </a>
    <a
      class="ml-2 hover:underline"
      href="http://www.google.com"
    >
      Discord server
    </a>
  </footer>
</div>
`;
